app/views/user_mailer/new_device_sign_in.html.erb

```diff
@@ -18,13 +18,15 @@
 
 <p>
   <%= t(
-        'user_mailer.new_device_sign_in.info_html',
+        'user_mailer.new_device_sign_in.info',
         app_name: APP_NAME,
-        date: @login_date,
-        location: @login_location,
-      ) %>
+      ) %>    
 </p>
 
+<br/>
+
+<p><strong><%= @login_date %><br/><%= @login_location %>, <%= @device_name %></strong></p>
```

Comment:
Since the date and location were separated by a line break before, I think I assumed that device name would similarly be on its own line, since I don't know that location and device name are linked strongly. Could also check with UX if we want some feedback?

```suggestion
<p><strong><%= @login_date %><br/><%= @login_location %><br/><%= @device_name %></strong></p>
```

Reply:
Synced on Slack, there's a difference between real emails and the preview mailer where real emails are formatted differently and prefix with "From [IP] (address potentially located in [location], [device name]", and the preview mailer instead is formatted simply as "[location], [device name]".

We should want to get those in sync, but I don't think it's quite as pressing an issue with the expanded formatting of the "From [...]", but we could double-check with UX folks to see if they have any thoughts on the phrasing.

Reply:
I'll bring up the the formatting issue of comma vs. line break vs. other options with UX.